types/lib/Modal.d.ts

```diff
@@ -25,6 +25,7 @@ export interface ModalProps extends React.HTMLAttributes<HTMLElement> {
   backdropTransition?: FadeProps;
   modalTransition?: FadeProps;
   centered?: boolean;
+  fullscreen?: boolean | 'sm' | 'md' | 'lg' | 'xl';
```

Comment:
Are we missing an `xxl` option?  Or is that just `true`?

Reply:
we haven't added `xxl` to anything yet. This is probably the only conflict with those changes, so might as well get this PR in first and can go back and add `xxl` to it in that change.

Reply:
I also didn't see the xxl option in the modal documentation 🤔

Reply:
oh yeah, there doesn't appear to be an `xxl` [modal size](https://v5.getbootstrap.com/docs/5.0/components/modal/#fullscreen-modal)